libiop/profiling/boost_profile.cpp

```diff
@@ -0,0 +1,43 @@
+#ifndef CPPDEBUG /* Ubuntu's Boost does not provide binaries compatible with libstdc++'s debug mode so we just reduce functionality here */
+#include <boost/program_options.hpp>
+#endif
+
+#include "libiop/bcs/bcs_common.hpp"
+
+namespace po = boost::program_options;
+using namespace libiop;
+
+typedef struct options{
+    std::size_t log_n_min = 8;
+    std::size_t log_n_max = 20;
+    std::size_t security_level = 128;
+    std::size_t field_size = 181;
+    std::size_t hash_enum_val = (size_t) libiop::blake2b_type;
+    bool heuristic_ldt_reducer_soundness = true;
+    bool is_multiplicative = true;
+    bool make_zk = false;
+    libiop::bcs_hash_type hash_enum = blake2b_type;
+} options;
```

Comment:
I don't think this trailing `options` is necessary. ref member initializers in: https://www.fluentcpp.com/2018/06/15/should-structs-have-constructors-in-cpp/

Reply:
Oops my bad, I think I used C struct syntax by accident. Should be fixed in the latest commit.